src/views/forms/NewMarketForm/index.tsx

```diff
@@ -31,22 +31,29 @@ import { NewMarketSelectionStep } from './NewMarketSelectionStep';
 import { NewMarketSuccessStep } from './NewMarketSuccessStep';
 import { NewMarketPreviewStep as NewMarketPreviewStep2 } from './v7/NewMarketPreviewStep';
 import { NewMarketSelectionStep as NewMarketSelectionStep2 } from './v7/NewMarketSelectionStep';
+import { NewMarketSuccessStep as NewMarketSuccessStep2 } from './v7/NewMarketSuccessStep';
 
 export enum NewMarketFormStep {
   SELECTION,
   PREVIEW,
   SUCCESS,
 }
 
+type NewMarketFormProps = {
+  defaultLaunchableMarketId?: string;
+  onSuccessAction?: () => void;
+  setFormStep?: Dispatch<SetStateAction<NewMarketFormStep | undefined>>;
```

Comment:
do we need to use Dispatch<SetStateAction>> for these? Normal practice is to just type them as normal functions and that's typically easier to read.

Reply:
I tend to lean towards leaving the `Dispatch<SetStateAction>` type. For myself, it is a quick indicator on whether the prop is stable and can be used in hook deps. Can change if you feel really strongly about this.